recipe-client-addon/lib/PreferenceExperiments.jsm

```diff
@@ -285,6 +285,15 @@ this.PreferenceExperiments = {
   },
 
   /**
+   * Get a list of all stored experiment objects.
+   * @resolves {Experiment[]}
+   */
+  async getAll() {
+    const store = await ensureStorage();
+    return Object.values(store.data).map(experiment => Object.assign({}, experiment));
```

Comment:
It is, I thought I had a comment here. Must've forgot, whoops!

The typedef for experiment objects at the top explicitly calls out the one-level-deep requirement, so it's better than not being documented. We don't use it, but I also like that cloning with `Object.assign` is possible within sandboxes as well as outside of them, vs `cloneInto`, which would require writing a deep copy for cloning outside of chrome code.

I don't really feel strongly one way or another. You think we might as well use `cloneInto`?